frontend/components/SetupPage/SetupYourAgent/SetupYourAgent.tsx

```diff
@@ -229,6 +262,22 @@ const SetupYourAgentForm = () => {
 };
 
 export const SetupYourAgent = () => {
+  const { selectedAgentType } = useServices();
+  const serviceTemplate = SERVICE_TEMPLATES.find(
+    (template) => template.agentType === selectedAgentType,
+  );
+
+  if (!serviceTemplate) {
```

Comment:
wonder if it is possible to be in such a situation, where we don't have the agent type selected...
maybe we can just get the serviceTemplate right inside the SetupYourAgentForm, I don't think we'll use this form anywhere else